README.md

```diff
@@ -1 +1,294 @@
+  root: [Circular],
+  Buffer:
+   { [Function: Buffer]
+     poolSize: 8192,
+     isBuffer: [Function: isBuffer],
+     compare: [Function: compare],
+     isEncoding: [Function],
+     concat: [Function],
+     byteLength: [Function: byteLength] },
+  clearImmediate: [Function],
+  clearInterval: [Function],
+  clearTimeout: [Function],
+  setImmediate: [Function],
+  setInterval: [Function],
+  setTimeout: [Function],
+  console: [Getter],
+  module:
+   Module {
+     id: 'repl',
+     exports:
+      { writer: [Object],
+        _builtinLibs: [Object],
+        REPLServer: [Object],
+        REPL_MODE_SLOPPY: Symbol(repl-sloppy),
+        REPL_MODE_STRICT: Symbol(repl-strict),
+        REPL_MODE_MAGIC: Symbol(repl-magic),
+        start: [Function],
+        repl: [Object] },
+     parent: undefined,
+     filename: '/Users/azat/Documents/Code/learn-co/node-debugger/repl',
+     loaded: false,
+     children: [],
+     paths:
+      [ '/Users/azat/Documents/Code/learn-co/node-debugger/repl/node_modules',
+        '/Users/azat/Documents/Code/learn-co/node-debugger/node_modules',
+        '/Users/azat/Documents/Code/learn-co/node_modules',
+        '/Users/azat/Documents/Code/node_modules',
+        '/Users/azat/Documents/node_modules',
+        '/Users/azat/node_modules',
+        '/Users/node_modules',
+        '/node_modules' ] },
+  require:
+   { [Function: require]
+     resolve: [Function],
+     main: undefined,
+     extensions: { '.js': [Function], '.json': [Function], '.node': [Function] },
+     cache: {} },
+  _: [Circular],
+```
+
+Let's cover the most important of them in more details.
+
+Note: Another name for `global` is `GLOBAL`. They are aliases. Also, `global.user` and `GLOBAL.user` are the same.
+
+## Global Object Properties
+
+So what are the Node-specific global interfaces? Here's the list of Node global interfaces which do not exist in browser JavaScript:
+
+* `process`: Contains the information about the system and environment, e.g., node version (`process.versions.node`)
+* `module`: Exports the functionality to be used by other programs/files, i.e., utilized to create modules
+* `exports`: Exports the functionality to be used by other programs. Alias of `module.exports`.
+* `require`: Imports the functionality from another file/module/package, i.e., utilizes modules
+* `Buffer`: Binary data type 
+* `__dirname`: Path and folder name of the current script
+* `_filename`: Filename of the current script 
+
+Know that `global` object has interfaces specific to Node. There are interfaces that are the same as in browser JavaScript:
+
+* `setInterval()`
+* `setTimeout()`
+* `console`
+* `clearTimeout(t)`
+* `clearInterval(t)`
+
+Note: Developers you can access them with or without referring to the `global` object. For example, both `global.process` and `process` work the same. 
+
+`Process`, `modules` and `Buffer` require their own lessons, so we'll cover them one-by-one later. For now let's look more into `process`. 
```

Comment:
are we going into process more here? the scoping section directly below doesn't go into process at all. not sure what `process` is at this point